superset/config.py

```diff
@@ -326,6 +326,7 @@ def _try_json_readsha(filepath, length):  # pylint: disable=unused-argument
 # The file upload folder, when using models with files
 UPLOAD_FOLDER = BASE_DIR + "/app/static/uploads/"
 UPLOAD_CHUNK_SIZE = 4096
+UPLOAD_MAX_SIZE = 50 * 1024 * 1024
```

Comment:
NIT: how about `UPLOAD_MAX_BYTES` to clarify the unit

Reply:
Thank you very much for your suggestion, it has been adopted